src/RokuDeploy.spec.ts

```diff
@@ -8,14 +8,15 @@ import * as path from 'path';
 import * as JSZip from 'jszip';
 import * as child_process from 'child_process';
 import * as glob from 'glob';
-import { RokuDeploy } from './RokuDeploy';
 import * as errors from './Errors';
 import { util, standardizePath as s, standardizePathPosix as sp } from './util';
 import type { FileEntry, RokuDeployOptions } from './RokuDeployOptions';
 import { cwd, expectPathExists, expectPathNotExists, expectThrowsAsync, outDir, rootDir, stagingDir, tempDir, writeFiles } from './testUtils.spec';
 import { createSandbox } from 'sinon';
 import * as r from 'postman-request';
 import type * as requestType from 'request';
+import { RokuDeploy } from './RokuDeploy';
+// eslint-disable-next-line no-duplicate-imports
```

Comment:
Hmm. we shouldn't need this eslint disable comment. All of our other projects support 1 `import` and 1 `import type`, so not sure what's different about this project. 
(see this example from brighterscript)
![image](https://github.com/rokucommunity/roku-deploy/assets/2544493/c51f569c-06a3-4719-904a-a04b62d7ecfc)

Reply:
Followed guidance from https://github.com/typescript-eslint/typescript-eslint/issues/2315 and used eslint plugin import